src/08_comprehensions.py

```diff
@@ -10,14 +10,14 @@
 
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
-y = []
+y = [x + 1 for x in range(5)]
 
-print (y)
+print(y)
 
 # Write a list comprehension to produce the cubes of the numbers 0-9:
 # [0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
 
-y = []
+y = [x * x for x in range(9)]
```

Comment:
this is a good spot for python's exponent syntax. So, x ** 3. You're good though.

Reply:
Oohhh. I was dumb and my brain kept reading it as squared rather than cubed. But thank you!